hedera-node/hedera-app/src/xtest/java/contract/XTestConstants.java

```diff
@@ -149,4 +130,8 @@ public static void addErc20Relation(
     public static Consumer<org.apache.tuweni.bytes.Bytes> assertSuccess() {
         return output -> assertEquals(SUCCESS_AS_BYTES, output);
     }
+
+    public static Consumer<org.apache.tuweni.bytes.Bytes> assertSuccess(String failureMessage) {
```

Comment:
Perhaps rename this (instead of overloading) because it's kind of weird to have `assertSuccess` with a parameter that is `failureMessage`.  Maybe `assertSuccessWithMessage()`?  Or maybe just rename the parameter to simply `message`?

Reply:
Renamed parameter to `orElseMessage`, so `assertSuccess(orElseMessage)`. 🤔